ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -305,6 +307,38 @@ - (void)errorViewDidSelectRetry:(EXErrorView *)errorView
 
 - (UIInterfaceOrientationMask)supportedInterfaceOrientations
 {
+  NSArray *legacy = @[@"33.0.0", @"34.0.0", @"35.0.0"];
```

Comment:
Can we make it more smart to not have an array of legacy SDK versions? It would be better to check if the current SDK version is lower or equal to `35.0.0` instead.

Reply:
Is there a method available just to make this comparison? Cause I can't see any in the code.

Reply:
There are several different building blocks that could work. Eg.

https://github.com/expo/expo/blob/c45d68862869efec0ac4da28c53f79961204c05f/ios/Exponent/Kernel/ReactAppManager/EXReactAppManager.m#L169

shows how to compute current SDK version. 🤔 https://github.com/expo/expo/blob/master/ios/Exponent/Kernel/Environment/EXVersions.h provides different methods for accessing SDK versions available.